File: business/src/main/java/work/beltran/discogsbrowser/business/WantedInteractor.java
package work.beltran.discogsbrowser.business;

import rx.Observable;
import work.beltran.discogsbrowser.api.model.UserWanted;

/**
 * Created by Miquel Beltran on 8/27/16
 * More on http://beltran.work
 */
public interface WantedInteractor {
    Observable<UserWanted> getWanted(int page);
}

File: business/src/main/java/work/beltran/discogsbrowser/business/WantedInteractor.kt
package work.beltran.discogsbrowser.business

import io.reactivex.Single
import work.beltran.discogsbrowser.api.model.UserWanted

/**
 * Created by Miquel Beltran on 8/27/16
 * More on http://beltran.work
 */
interface WantedInteractor {
    fun getWanted(page: Int): Single<UserWanted>
}
